src/main/java/no/nav/veilarbaktivitet/domain/Person.java

```diff
@@ -25,6 +25,14 @@ public static NavIdent navIdent(String navIdent) {
         return new NavIdent(navIdent);
     }
 
+    public boolean isExtern() {
```

Comment:
Forstår ikke helt hvorfor dette er nødvendig. Vi pleier å bruke authContextHolder.erInternBruker() for å skille mellom bruker typer

Reply:
Nå brukes denne kun internt i klassen, så den kunne vært private, men jeg mener denne er nyttig i flere sammenhenger. Bruk av authContextHolder threadlocal er etter min mening et ganske alvorlig antipattern. Når vi har mulighet for en mer 'funksjonell' approach, så bør vi bruke det.